src/Pact/Native/Guards.hs

```diff
@@ -222,7 +222,7 @@ typeOfPrincipalDef :: NativeDef
 typeOfPrincipalDef = defRNative "typeof-principal" typeOfPrincipal
   (funType tTyString [("principal", tTyString)])
     [LitExample
-     "(typeof-principal 'k:462e97a099987f55f6a2b52e7bfd52a36b4b5b470fed0816a3d9b26f9450ba69)"]
```

Comment:
These are actually going to be forking changes - hang tight on this PR - updating native docs is more complex than it seems, since they go on chain. We'll raise a ticket internally to work on this PR.